kubernetes/templates/astaire-depl.tmpl

```diff
@@ -28,13 +29,17 @@ spec:
         livenessProbe:
           tcpSocket:
             port: 11311
-          initialDelaySeconds: 30
+          initialDelaySeconds: 90
```

Comment:
Is this to avoid the fact that astaire containers often hit a restart on first boot? 
Seems like quite a long wait, given you were annoyed at things taking a minute or more to spin up before

Reply:
Yes.   The container doesn't report that it is alive until it has clustered I think and if clustering takes too long then we kill it off and restart it which can lead to the cluster ending up broken.   After conversation yesterday  I've now increased the number of retries instead.